packages/plugins/blocks/blocks-basic/src/blocks/Anchor/Anchor.js

```diff
@@ -15,28 +15,9 @@
 */
 
 import React from 'react';
-import { get, type } from '@lowdefy/helpers';
+import { get } from '@lowdefy/helpers';
 import { blockDefaultProps } from '@lowdefy/block-utils';
 
-const Strong = ({ children, strong }) => (strong ? <b>{children}</b> : <>{children}</>);
-const Tag = ({ blockId, children, className, disabled, href, Link, newTab, onClick, rel }) =>
-  disabled ? (
-    <span id={blockId} className={className}>
-      {children}
-    </span>
-  ) : (
-    <Link
-      id={blockId}
-      className={className}
-      href={href}
-      onClick={onClick}
-      rel={rel || 'noopener noreferrer'}
```

Comment:
I assume these are set properly by `next/link`?

Reply:
createLink handles them where applicable, but should probably be configurable.